packages/backend/Cargo.toml

```diff
@@ -8,7 +8,7 @@ exclude = ["/bindings", "/api-wrapper"]
 [dependencies]
 tokio = { version = "1.12.0", default-features = false }
 once_cell = { version = "1.8.0", default-features = false }
-iota-wallet = { git = "https://github.com/iotaledger/wallet.rs", rev = "a084002da15c072f0f7256c74403b05dad3c7d8b", default-features = false, features = ["stronghold", "ledger-nano", "ledger-nano-simulator", "participation"] }
+iota-wallet = { git = "https://github.com/iotaledger/wallet.rs", rev = "7f2529e6af1560436793e45578f5a161a77a7617", default-features = false, features = ["stronghold", "ledger-nano", "ledger-nano-simulator", "participation"] }
```

Comment:
```suggestion
iota-wallet = { git = "https://github.com/iotaledger/wallet.rs", rev = "d9a74bc6f8996160c7c86097963d8082b21b1bc8", default-features = false, features = ["stronghold", "ledger-nano", "ledger-nano-simulator", "participation"] }
```

Reply:
I rebased the voting branch to include the changed from dev (mainly signature validation, which will prevent invalid transactions for ledger users, when they use a wrong 25th word/ wrong ledger)